### PR TITLE
R.I.P. LESS

### PR DESCRIPTION
**Pull request by DanKGooGLy**

### CODE REVIEW

File: source/styles/base.css [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+}
+.pfp {
+	margin-right: 6px;
+	height: 48px;
+	width: 48px;
+	border-radius: 100%;
+	background: #58a5f0;
+	border: 2px solid #0277bd;
+}
+
```
Comment (BBaoVanC) [inline source/styles/base.css:119]: missing trailing newline

Comment (DanKGooGLy) [reply source/styles/base.css:119]: am i supposed to yeet that or

Comment (BBaoVanC) [reply source/styles/base.css:119]: you are supposed to add the trailing newline, most editors automatically do it

**Comment by mugman174** [comment]
lemme check

**Comment by mugman174** [comment]
gtg

**Pull request merged**